scene/gui/color_picker.cpp

```diff
@@ -857,10 +955,14 @@ void ColorPicker::add_preset(const Color &p_color) {
 		_add_preset_button(_get_preset_size(), p_color);
 	}
 
+	palette_name->set_text(vformat(ETR("%s*"), palette_name->get_text().replace("*", "")));
```

Comment:
Don't think this needs translation
```suggestion
	palette_name->set_text(vformat("%s*", palette_name->get_text().replace("*", "")));
```